src/EfficientDynamoDb.Credentials.AWSSDK/EfficientDynamoDb.Credentials.AWSSDK.csproj

```diff
@@ -26,9 +26,9 @@
     </ItemGroup>
 
     <ItemGroup>
-      <PackageReference Include="AWSSDK.Core" Version="3.7.0.13" />
-      <PackageReference Include="EfficientDynamoDb" Version="0.9.2-alpha.0.4" />
-      <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1">
+      <PackageReference Include="AWSSDK.Core" Version="3.7.303.28" />
+      <PackageReference Include="EfficientDynamoDb" Version="0.9.15" />
```

Comment:
Should we reference the package from local instead of a versioned package here? or doesn't matter?